platform/remarkable-aarch64/external.manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "KOReader",
+  "application": "reader.lua",
+  "environment": {
+    "LD_PRELOAD": "/home/root/shims/qtfb-shim.so",
+    "QTFB_SHIM_MODEL": "false",
+    "QTFB_SHIM_INPUT_MODE": "RMPP",
+    "QTFB_SHIM_MODE": "RGB565",
+    "KO_DONT_GRAB_INPUT": "1"
+  },
+  "qtfb": true
+}
```

Comment:
What's that for? Is `koreader.sh` still used?

Reply:
It's for AppLoad. (launcher for modern reMarkable firmware) `koreader.sh` is not used for modern firmware (and not at all for rMPP) but kept for legacy use.

Reply:
So no OTA updates?

Reply:
OTA should still work but can't test it.

Reply:
What's the problem with `koreader.sh`, why do you want to remove it?

Without it, you loose:
- `crash.log`
- restarting from KOReader
- installing OTA updates
- …

Reply:
I don't understand: fbink should already be compiled automatically, no?

Reply:
FBInk seems to be compiled in AArch32 thus not working. Should I check koreader-base?